src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -347,5 +375,10 @@ public EmitOptions WithIncludePrivateMembers(bool value)
 
             return new EmitOptions(this) { IncludePrivateMembers = value };
         }
+
+        public EmitOptions WithInstrument(string instrument)
+        {
+            return new EmitOptions(this) { Instrument = instrument };
```

Comment:
Please follow pattern for other similar methods, i. e. check if we can simply return ```this```.